pkg/cronjob/es_index_cleaner.go

```diff
@@ -45,6 +45,11 @@ func CreateEsIndexCleaner(jaeger *v1.Jaeger) *batchv1beta1.CronJob {
 
 	ca.Update(jaeger, commonSpec)
 
+	priorityClassName := ""
+	if jaeger.Spec.Storage.EsIndexCleaner.PriorityClassName != "" {
```

Comment:
Agree on this comment.

Reply:
Actually I looked at other parts of the code where priorityClass has been reference and this was the general approach followed elsewhere as well. Having to go with the same flow, I implemented the same set of changes

ref: 
1. https://github.com/jaegertracing/jaeger-operator/blob/main/pkg/deployment/agent.go#L92
2. https://github.com/jaegertracing/jaeger-operator/blob/main/pkg/deployment/query.go#L89

Reply:
Let me know if you guys wish to have the assignment in the metav1 object itself so that I can make the changes. Also let me know if you guys wish to have the same set of changes made at other places as well. If that be the case, should I do it in the same PR or create a new PR altogether??

Reply:
@rubenvp8510 / @frzifus : Guys, can you please share your inputs over it?

Reply:
Sorry for the long response. From my point of view it would make sense to change it at least in this pr. But feel free to change it on the other places too in this or a new pr. What do you think?

Reply:
Sure. Will make the necessary change only for the priorityClass which is part of this PR. 
And would create a new PR for the other 2 PriorityClass.

Reply:
I would say only do the change related to this PR, we can do other changes in a separate PR

Reply:
still a bit confused, isnt it the same like?
```suggestion
	priorityClassName := "jaeger.Spec.Storage.EsIndexCleaner.PriorityClassName"
```

up to you @rubenvp8510

Reply:
I agree with the change, but for not blocking this PR we can do the changes in a follow up PR.